Client/Assets/AllFramesAtDocumentEnd.js

```diff
@@ -1 +1 @@
),s=i.length>>>0;if("function"!=typeof n)throw new TypeError(n+" is not a function");arguments.length>1&&(t=e);r=new Array(s),o=0;for(;o<s;){var a,u;o in i&&(a=i[o],u=n.call(t,a,o,i),r[o]=u),o++}return r}(s,function(e){return{hostname:e.hostname,formSubmitURL:e.formSubmitURL,httpReal:e.httpRealm,username:e.username,password:e.password,usernameField:e.usernameField,passwordField:e.passwordField}})).filter(function(e){var t=e.username.length<=d&&e.password.length<=c;return t||n("Ignored",e.username),t},this);if(0==m.length)return[!1,s];if(f.value&&!o)return"existingPassword",[!1,s];var g=!1;!r&&(this._isAutocompleteDisabled(e)||this._isAutocompleteDisabled(u)||this._isAutocompleteDisabled(f))&&(g=!0,n("form not filled, has autocomplete=off"));var v=null;if(u&&(u.value||u.disabled||u.readOnly)){var p=u.value.toLowerCase();if((w=m.filter(function(e){return e.username.toLowerCase()==p})).length){for(var _=0;_<w.length;_++){var h=w[_];h.username==u.value&&(v=h)}v||(v=w[0])}else"existingUsername",n("Password not filled. None of the stored logins match the username already present.")}else if(1==m.length)v=m[0];else{var w;v=(w=u?m.filter(function(e){return e.username}):m.filter(function(e){return!e.username}))[0]}var b=!1;if(v&&t&&!g){if(u){var y=u.disabled||u.readOnly,F=v.username!=u.value,L=i&&F&&u.value.toLowerCase()==v.username.toLowerCase();y||L||!F||(u.value=v.username,l(u,"keydown",40),l(u,"keyup",40))}f.value!=v.password&&(f.value=v.password,l(f,"keydown",40),l(f,"keyup",40)),b=!0}else v&&!t?("noAutofillForms",n("autofillForms=false but form can be filled; notified observers")):v&&g&&("autocompleteOff",n("autocomplete=off but form can be filled; notified observers"));return[b,s]}},o={_getPasswordOrigin:function(e,n){return e},_getActionOrigin:function(e){var n=e.action;return""==n&&(n=e.baseURI),this._getPasswordOrigin(n,!0)}};function i(e){t.onUsernameInput(e)}var s=document.body;function a(e){for(var n=0;n<e.length;n++){var t=e[n];"FORM"===t.nodeName?u(t):t.hasChildNodes()&&a(t.childNodes)}return!1}function u(e){try{t._asyncFindLogins(e,{}).then(function(e){t.loginsFound(e.form,e.loginsFound)}).then(null,n)}catch(e){n(e)}}function l(e,n,t){var r=document.createEvent("KeyboardEvent");r.initKeyboardEvent(n,!0,!0,window,0,0,0,0,0,t),e.dispatchEvent(r)}new MutationObserver(function(e){for(var n=0;n<e.length;++n)a(e[n].addedNodes)}).observe(s,{attributes:!1,childList:!0,characterData:!1,subtree:!0}),window.addEventListener("load",function(e){for(var n=0;n<document.forms.length;n++)u(document.forms[n])}),window.addEventListener("submit",function(e){try{t._onFormSubmit(e.target)}catch(e){n(e)}}),Object.defineProperty(window.__firefox__,"logins",{enumerable:!1,configurable:!1,writable:!1,value:Object.freeze(new function(){this.inject=function(e){try{t.receiveMessage(e)}catch(e){}}})})})},function(e,n,t){"use strict";window.__firefox__.includeOnce("PrintHandler",function(){window.print=function(){webkit.messageHandlers.printHandler.postMessage({})}})}]);
```

Comment:
I think you have to remove this change from the PR, otherwise our CI throws an error.
This line sometimes get changed when running the bootstrap script, sorry about that

Reply:
I forgot to include `ContextMenu.js` in the commit. Fixed.